.github/workflows/copywrite.yml

```diff
@@ -0,0 +1,28 @@
+name: Copywrite
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  copywrite:
```

Comment:
Can you add this job to the ci.yml workflow instead? What defines a workflow most of all is when it is run, and in this case we want this to run in CI events

Reply:
Done. I'll update the other repos as well.